src/commons/achievement/control/editorTools/editableUtils/goals/EditableAchievementGoal.tsx

```diff
@@ -1,3 +1,10 @@
+import React from 'react';
+
+function EditableAchievementGoal() {
+  return <></>;
+}
```

Comment:
What is this for?

Reply:
This is part of upcoming changes in the next PR (same as below)